### PR TITLE
Define a center piece

### PR DESCRIPTION
**Pull request by sam596**
As discussed in #710

I added @lgarron's suggestion, adding in a "defined as".

### CODE REVIEW

**Comment by xsrvmy** [comment]
I think we need to be a little more careful that someone does not argue that a split part of a corner/wing/edge piece is a center piece using this definition

**Comment by sam596** [comment]
Hmm, I see what you mean, but I think the 5x5 example negates that.

**Comment by lgarron** [comment]
> Hmm, I see what you mean, but I think the 5x5 example negates that.

Yeah, I think this is pragmatic enough.

**Comment by lgarron** [comment]
@sam596: Would you be alright with removing "defined as"? I specifically wanted to avoid defining what a center piece is, since this doesn't generalize to all puzzles.

File: wca-guidelines.md [hunk 1]
```diff
@@ -80,6 +80,7 @@ To be more informative, each Guideline is classified using one of the following
 - 3k2b++) [EXAMPLE] Suppose the competitor attempts 10 puzzles and solves 8 successfully. If it is found that two solved puzzles and one unsolved puzzle are not permitted, the competitor is considered to have a result with 6 solved puzzles out of 10 (rather than DNF), at the discretion of the WCA Delegate.
 - 3l+) [ADDITION] Logos may feature any reasonable design that does not give a conspicuous advantage (e.g. encoding information that could be used to cheat), and leave the colored part clearly recognizable. Unconventional logos are only permitted at the discretion of the WCA Delegate.
 - 3l++) [CLARIFICATION] In the past, puzzles with logos have been permitted for blindfolded events. Such puzzles are no longer permitted.
+- 3l1+) [CLARIFICATION] On NxNxN puzzles, a center piece is defined as any piece with a single colored part. For example, the 5x5x5 Cube has 9 center pieces on each face.
```
Comment (lgarron) [inline wca-guidelines.md]:
```suggestion
- 3l1+) [CLARIFICATION] On NxNxN puzzles, a center piece is any piece with a single colored part. For example, the 5x5x5 Cube has 9 center pieces on each face.
```

Comment (sam596) [reply wca-guidelines.md]: I just think it sounded better, but sure, if you think that's best.

**Pull request merged**